app/Resources/views/events/show.html.twig

```diff
@@ -21,7 +21,7 @@
     <div class="container">
         <div class="page-header">
             <h1>
-                {{ event.displayTitle }}
+                {{ bdi(event.displayTitle) }}</bdi>
```

Comment:
I'm guessing the `</bdi>` shouldn't be there?

Reply:
Whoops that's a remnant of an experiment. Thanks for catching that!